opensquirrel/squirrel_ir.py

```diff
@@ -7,10 +7,8 @@
 from typing import Callable, List, Optional, Tuple
 
 import numpy as np
-
```

Comment:
@elenbaasc, @pablolh Are we enforcing PEP styles somehow? I know CLion tell you if you need to add or remove newlines, I suppose PyCharm will do too. If @juanboschero is using VSCode, I understand you can use a plugin to do these checks for you.

Reply:
I think you need to run `isort` and `black` to fix the code in order to pass static analysis in the CI

Agreed that my PyCharm tells me to use 2 blank lines between top level functions/classes

Reply:
Yes, I am currently using VSCode with pylint checks however, for some reason, it recommends place a new line between the imports and class. I will need to check the PEP compliance I am using.